apps/test/unit/templates/projects/SectionProjectsListTest.js

```diff
@@ -8,45 +9,49 @@ const STUB_PROJECTS_DATA = [
     channel: 'ABCDEFGHIJKLM01234',
     name: 'Antelope Freeway',
     studentName: 'Alice',
-    type: 'applab',
-    updatedAt: '2016-12-31T23:59:59.999-08:00'
+    type: 'weblab',
+    updatedAt: '2016-12-29T11:00:00-00:00'
   },
   {
     channel: 'AAAABBBBCCCCDDDDEE',
     name: 'Cats and Kittens',
     studentName: 'Charlie',
     type: 'weblab',
-    updatedAt: '2016-11-30T00:00:00.001-08:00'
+    updatedAt: '2016-11-30T11:00:00-00:00'
   },
   {
     channel: 'NOPQRSTUVWXYZ567879',
-    name: 'Batyote',
+    name: 'Dominoes',
     studentName: 'Bob',
     type: 'gamelab',
-    updatedAt: '2017-01-01T00:00:00.001-08:00'
+    updatedAt: '2017-01-01T11:00:00-00:00'
   },
   {
     channel: 'VVVVWWWWXXXXYYYYZZ',
-    name: 'Another App',
+    name: 'A1 Locksmith',
     studentName: 'Alice',
     type: 'applab',
-    updatedAt: '2016-10-29T00:00:00.001-08:00'
+    updatedAt: '2016-10-29T11:00:00-00:00'
   },
 ];
 
 const STUDIO_URL_PREFIX = '//foo-studio.code.org';
 
-const PROJECT_NAME_COLUMN_INDEX = 0;
-const STUDENT_NAME_COLUMN_INDEX = 1;
-
 /**
  * @param {HTMLTableRowElement} rowElement HTML row element in the projects list table
  * @param {string} projectName Expected project name
  * @param {string} studentName Expected student name
+ * @param {string} appType Expected app type (App Lab, Game Lab, etc)
+ * @param {string} lastEdited Expected last edited date (Month DD, YYYY). Note that this
+ *   format is used only in unit tests due to incorrect date formatting in PhantomJS.
+ *   The desired date format which we will show in all browsers is MM/DD/YYYY.
  */
-function assertRowContents(rowElement, projectName, studentName) {
-  expect(rowElement.childNodes[PROJECT_NAME_COLUMN_INDEX].innerText).to.equal(projectName);
-  expect(rowElement.childNodes[STUDENT_NAME_COLUMN_INDEX].innerText).to.equal(studentName);
+function assertRowContents(rowElement, projectName, studentName, appType, lastEdited) {
+  expect(rowElement.childNodes[COLUMNS.PROJECT_NAME].innerText).to.equal(projectName);
+  expect(rowElement.childNodes[COLUMNS.STUDENT_NAME].innerText).to.equal(studentName);
+  expect(rowElement.childNodes[COLUMNS.APP_TYPE].innerText).to.equal(appType);
+  // Temporarily comment out this line to make tests pass locally in Chrome
+  expect(rowElement.childNodes[COLUMNS.LAST_EDITED].innerText).to.equal(lastEdited);
```

Comment:
Because `toLocaleDateString` returns dates like `MM/DD/YYYY` in Chrome and every other browser we support, while PhantomJS returns dates like `Month DD, YYYY`.